tests/modelTests.php

```diff
@@ -113,3 +113,35 @@ function getAllTombSectionLettersTest(){
 }
 
 
+function getBuriedIndividualsForColumbariumTest($columbariumID){
+    $response = getBuriedIndividualsForColumbarium($columbariumID);
+    if(count($response->result) > 0)
+        echo json_encode(get_object_vars($response ->result[0]));
+if(count($response -> error) > 0)
+        echo $response -> error[0];
+}
+
+
+
+function getColumbariumAttachmentsTest($columbariumId){
+    $response = getColumbariumAttachments($columbariumId);
+    if(count($response->result) > 0)
+        echo json_encode(get_object_vars($response ->result[0]));
+if(count($response -> error) > 0)
+        echo $response -> error[0];
+}
+
+function getAllColumbariumRelatedDataWithFilterTest()
```

Comment:
This test method probably did not return any niche types or columbarium types with the fetched columbarium. This is simply because the method `getColumbariumRelatedDataWithFilter` did not call the setters of the Columbarium object. After you add your changes, make sure you test every filter parameter and make sure that all of the associated data is returned.